use-cases/rag-pipeline/alloy-db-setup/src/create_catalog.py

```diff
@@ -54,20 +42,20 @@ def create_database(database, new_database):
             ) as connection:
                 with connection.begin():
                     connection.execute(del_db)
-                    logger.info(f"Database '{new_database}' deleted successfully.")
+                    logging.info(f"Database '{new_database}' deleted successfully.")
```

Comment:
Why did you switch these back to logging?

Reply:
Thankyou . I must have switched while debugging something.